src/features/Main/Sidebar/index.scss

```diff
@@ -2,21 +2,31 @@
 @import "assets/variables.scss";
 
 .nav-links {
+  list-style-type: none;
+  margin: 20px 0;
+  padding: 0;
+
   li {
-    button {
-      align-items: center;
-      border-radius: 16px;
-      color: $color-gray;
-      display: flex;
-      font-size: 18px;
-      gap: 16px;
-      justify-content: flex-start;
-      line-height: 24px;
-      padding: 8px 12px;
-      text-decoration: none;
-      background: transparent;
-      border: none;
-      width: 100%;
+    padding: 4px 0;
+  }
```

Comment:
Please, add a blank line after this line to separate rules. @AuraAlba

Reply:
done